### DISCUSSION TITLE
allure-results don't upload from Jenkins run

### DISCUSSION THREAD

**Discussion by alexanderyellow** [allure-framework/allure2#1365] [TestOps]
Hello! I have gradle project with the several modules. I have Jenkinsfile.groovy file with the following stage:

```
stage("Build and execute tests") {
            def testsGradleParam = " --tests com.company.CompanyRegistrationTestSuite"

            java.inside {
                withAllureUpload(serverId: 'allure-dev', projectId: '1', results: [[path: 'qa-automation/test-server/build/allure-results']]) {
                    dir('core/gradle') {
                        withCredentials([string(credentialsId: 'credential', variable: 'token')]) {
                            sh("rm -f results && ./gradlew :test-server:clean && ./gradlew :test-server:testFunctional -Dtoken=$token  $testsGradleParam | tee results")
                        }
                    }
                }
            }
        }
```

Tests passed successfully, allure-results folder is created in the qa-automation/test-server/build/allure-results, Allure report generated successfully (uses the same path),  launch in TestOps created but results don't upload to this launch.

In the Jenkins pipeline run there are following logs:
13:43:13  [Allure] [66712] Job run [http://allure-ee-gateway.allure:8080/jobrun/56] started....
13:43:14  [Allure] [66712] Watching on [qa-automation/test-server/build/allure-results] for session [54]

...

13:44:52  [Allure] [66865] Session [54] total indexed [9], ignored [0], processed [0], errors [9]
13:44:52  [Allure] [66865] Session [54] stopped successfully...
13:44:52  [Allure] [66865] Job run [http://allure-ee-gateway.allure:8080/jobrun/56] stopped successfully...

Could you help me please understand what goes wrong and how to fix it? Maybe I can see TestOps logs somewhere.

**Comment by cheshi-mantu** [comment]
Hi, all the issues related to Allure TestOps must be submitted through different channel as per discussions policy (#1296). Please file your request registered with corporate email address here: https://support.qameta.io
see you there.

**Reply by alexanderyellow** [reply]
Thank you.
